src/main/kotlin/vm/Opcode.kt

```diff
@@ -63,6 +63,9 @@ enum class Opcode(val argCount: Int = 0) {
     // Set range pop2..pop1 of value pop0.
     O_SETRANGE,
 
+    // Spread-assign pop1 list to pop2 list of varnames.
+    O_SPREAD,
```

Comment:
Usually spread means using the `...` operator to expand or create a list, so the terminology here threw me a bit.  You might consider naming this something else (group assign?) if you think you'll ever actually want a traditional spread operator.

Reply:
Ah shit thank you.  I think some part of me knew I was wrong here.